cwms-data-api/src/main/java/cwms/cda/data/dao/TimeSeriesGroupDao.java

```diff
@@ -65,7 +65,7 @@ public List<TimeSeriesGroup> getTimeSeriesGroups(String officeId, String groupOf
             whereCond = AV_TS_CAT_GRP.AV_TS_CAT_GRP.GRP_DB_OFFICE_ID.eq(officeId);
         }
 
-        return getTimeSeriesGroupsWhere(whereCond, groupOfficeId, categoryOfficeId);
+        return getTimeSeriesGroupsWhere(whereCond, officeId, groupOfficeId, categoryOfficeId);
```

Comment:
rename all "officeId" variables to "tsOfficeId" for clarity

Reply:
Renamed for clarity